authentication/views.py

```diff
@@ -37,25 +36,30 @@ def _keycloak_logout_url(self, user):
         qs = urlencode(
             {
                 "id_token_hint": id_token,
-                "post_logout_redirect_uri": settings.LOGOUT_REDIRECT_URL,
+                "post_logout_redirect_uri": self.request.build_absolute_uri(
```

Comment:
I can see this causing issues on a live server since self.request.build_absolute_uri will end up generating a different url depending on  if its hit from api.* (which it might if done via js) or the main hostname. I think we need to explicitly generate this from the APP_BASE_URL since redirecting back to api.mitopen.odl.mit.edu leaves the user in a broken state - some functionality does not work as intended from api.*

Reply:
The API should only ever be served from `api.*` domains. I updated the docstring to reflect the redirect path that the browser will take as the user is logged out.